src/briefcase/integrations/cookiecutter.py

```diff
@@ -61,8 +61,9 @@ def __init__(self, environment):
         super().__init__(environment)
 
         def escape_toml(obj):
-            """Escapes double quotes and backslashes."""
-            return obj.replace('"', '"').replace("\\", "\\\\")
+            """Escapes string to match toml string requirements
+            https://toml.io/en/         v1.0.0#string."""
```

Comment:
The original comment here is fine; there's no need to reference the spec.
```suggestion
            """Escapes double quotes and backslashes."""
```